module-db/Interface/ContactRecord.cpp

```diff
@@ -1559,3 +1559,14 @@ auto ContactRecordInterface::verifyTemporary(ContactRecord &record) -> bool
 
     return isTemporary;
 }
+
+auto ContactRecordInterface::isContactTemporary(const std::optional<ContactNumberHolder> &matchedNumber) -> bool
+{
+    if (matchedNumber.has_value()) {
```

Comment:
[minor]
You could invert the condition to `return` earlier making the code less indented and thus more readable